### PR TITLE
JHipster v4.5.2 (new formula)

### PR DESCRIPTION
**Pull request by jdubois**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/jhipster.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class Jhipster < Formula
+  desc "Generate, develop and deploy Spring Boot + Angular applications"
+  homepage "https://jhipster.github.io/"
+  url "https://registry.npmjs.org/generator-jhipster/-/generator-jhipster-4.5.2.tgz"
+  sha256 "5df0edbbdb685df5ae598b53fb2def43b6c23bb75d44763a63cda69567395954"
+
+  bottle :unneeded
```
Comment (ilovezfs) [inline Formula/jhipster.rb]: A bottle will be needed since it's building with npm, so you can just remove this.

File: Formula/jhipster.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"jhipster", "info"
```
Comment (ilovezfs) [inline Formula/jhipster.rb]: Please write a test that exercises its functionality.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/jhipster.rb (hunk 1 above)
Comment (jdubois) [inline Formula/jhipster.rb]: OK

File: Formula/jhipster.rb (hunk 2 above)
Comment (jdubois) [inline Formula/jhipster.rb]: I'm going to test the result of "jhipster info" - this is our own "audit" or "doctor" system, so if it says it's good, that should be OK

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
@jdubois Thanks for the new formula!

**Comment by jdubois** [comment]
Thank you so much for your help & review!!

**Pull request closed without merging**